project/build.properties

```diff
@@ -0,0 +1 @@
+sbt.version = 1.0.4
```

Comment:
:tada:

For people on osx, https://github.com/swoval/MacOSXWatchService is a must. Might be worth mentioning in contributing guide

Reply:
Looking at the source, it noops if your not on a mac, I see no reason to not add it to the project itself

Reply:
I never knew this by the way (I dev on a mac.... running NixOs)